lib/spree_avatax_certified/engine.rb

```diff
@@ -36,7 +36,9 @@ class Engine < Rails::Engine
 
     def self.activate
       Dir.glob(File.join(File.dirname(__FILE__), '../../app/models/**/*.rb')) do |c|
-        Rails.configuration.cache_classes ? require(c) : load(c)
+        # Some models raise "superclass mismatch" during "load(c)". Always use "require(c)".
+        # Rails.configuration.cache_classes ? require(c) : load(c)
+        require(c)
```

Comment:
This is a big step forward to know what is the exact problem. :+1: 
But I'm afraid of this is not a good solution. All of the rails and spree plugins are working like this: load the files in development mode instead of requiring. I think the main problem here the order of the loading. What do you think?